File: C++/Ladder.cpp
#include <iostream>
#include <math.h>

using namespace std;

int main() {
  int s1, deg;
  cin >> s1 >> deg;
  double length = s1 / sin(deg * 3.14159265358979 / 180);
  length = ceil(length);
  int out = (int) length;
  cout << out << endl;
}

File: Python/Ladder.py
import math

def main():
    l = input().split()
    s1 = int(l[0])
    deg = int(l[1])
    length = s1 / math.sin(deg * 3.14159265358979 / 180)
    length = math.ceil(length)
    ret = int(length)
    print(ret)
main()
